src/Scripting/CSharp/CSharpScript.cs

```diff
@@ -1,10 +1,13 @@
 // Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+#pragma warning disable RS0026 // Do not add multiple public overloads with optional parameters
```

Comment:
We should not be disabling this analyzer.  Need to address the problem and settle on a single overload with optional parameters.